### DISCUSSION TITLE
Looking out for CouchbaseStorageProvider for JobRunr

### DISCUSSION THREAD

**Discussion by amitghosh29021992** [jobrunr/jobrunr#504] [Q&A]
Hi All,

I am looking to use Couchbase as the storage provider for JobRunr. It would be really helpful if I can get some reference for the same. Thanks in advance!

**Comment by rdehuyss** [comment]
Well, take a look at the existing StorageProviders. As soon as you pass the StorageProviderTest, you know all is well :-)
